### DISCUSSION TITLE
What is the model.onnx file in .vscode/extensions folder?

### DISCUSSION THREAD

**Discussion by khanhvu207** [community/community#53672] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Just out of curiosity, I noticed that there is file named `model.onnx` in `/.vscode/extensions/github.copilot-1.83.41/dist`. Is it somehow the actual model being used? I think it's not the case since Github Copilot requires access to the Internet, and also the file is small, just around 8.4 megabytes.

**Comment by Akash1134** [comment, marked as answer]
Hi @khanhvu207 ,

The model.omnx file you're referring to is not the actual model. It is part of the Copilot engine that helps generate relevant topics and questions. 

Thanks!

**Reply by FinnChen** [reply]
In the latest version of github copilot, this `model.onnx` file has been removed.

**Comment by khanhvu207** [comment]
Hi, thank you for the clarification! For anyone who is also curious, I tried to load `model.onnx` file and was able to see the model's architecture:
<img width="499" alt="Screenshot 2023-05-03 at 14 51 28" src="https://user-images.githubusercontent.com/26273495/235921295-cee5bce3-5681-4a8c-846b-dfe4fad44c8d.png">
Looks like it's indeed a type of model for text classification. 😃
